service/view_building_state.hh

```diff
@@ -9,6 +9,9 @@
 
 #pragma once
 
+#include <seastar/core/sstring.hh>
+#include <stdexcept>
+
```

Comment:
Why do we need these headers? The commit in which they are introduced does not change anything else in this header.